contracts/test/veLON/Setup.t.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+import "forge-std/Test.sol";
+import "contracts/Lon.sol";
+import "contracts/veLON.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "contracts-test/utils/BalanceSnapshot.sol";
+
+contract TestVeLON is Test {
+    using BalanceSnapshot for BalanceSnapshot.Snapshot;
+    using SafeMath for uint256;
+
+    uint256 userPrivateKey = uint256(1);
+    uint256 otherPricateKey = uint256(2);
+
+    address user = vm.addr(userPrivateKey);
+    address other = vm.addr(otherPricateKey);
+
+    Lon lon = new Lon(address(this), address(this));
+    veLON veLon;
+
+    uint256 constant DEFAULT_STAKE_AMOUNT = 1e18;
+    uint256 constant MAX_LOCK_TIME = 365 days;
+    uint256 public constant PENALTY_RATE_PRECISION = 10000;
+
+    uint256 public earlyWithdrawPenaltyRate;
+
+    //record the balnce of Lon and VeLon in VM
+    BalanceSnapshot.Snapshot public stakerLon;
+    BalanceSnapshot.Snapshot public lockedLon;
+
+    function setUp() public {
+        // Setup
+        veLon = new veLON(address(lon));
+
+        // Deal 100 ETH to user
+        deal(user, 100 ether);
+        // Mint LON to user
+        lon.mint(user, 100 * 1e18);
+        // User approve LONStaking
+        vm.prank(user);
+        lon.approve(address(veLon), type(uint256).max);
+
+        // Deal 100 ETH to otherUSer
```

Comment:
`otherUser` ?

Reply:
Other user means the second user to interact with. I have added more comments to explain it.